teamcity: failed tests on master: acceptance/TestDecommission, acceptance/TestDecommission/runMode=local, acceptance/TestDecommission, acceptance/TestDecommission/runMode=local

The following tests appear to have failed:

[#497154](https://teamcity.cockroachdb.com/viewLog.html?buildId=497154):


```
--- FAIL: acceptance/TestDecommission (57.670s)
None
--- FAIL: acceptance/TestDecommission/runMode=local (57.670s)
test_log_scope.go:81: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestDecommission_runMode=local558569193
test_log_scope.go:62: use -show-logs to present logs inline
decommission_test.go:343: csv input:
	id,is_live,gossiped_replicas,is_decommissioning,is_draining
	1,false,20,true,false
	# 1 row
	id,is_live,gossiped_replicas,is_decommissioning,is_draining
	1,true,0,true,false
	# 1 row
	
	All target nodes report that they hold no more data. Please verify cluster health before removing the nodes.
	
	expected:
	[][]string{
	    {"id", "is_live", "gossiped_replicas", "is_decommissioning", "is_draining"},
	    {"1", "true", "0", "true", "true"},
	    {"# 1 row"},
	    {"All target nodes report that they hold no more data. Please verify cluster health before removing the nodes."},
	}
	errors:<nil>
	row #2, col #5: found "false" which does not match "true"
------- Stdout: -------
CockroachDB node starting at 2018-01-29 20:11:48.853669949 +0000 UTC (took 0.7s)
build:               CCL v2.0-alpha.20180129-76-g7b3bf98 @ 2018/01/29 20:06:00 (go1.9.3)
admin:               http://127.0.0.1:42815
sql:                 postgresql://root@127.0.0.1:41853?application_name=cockroach&sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestDecommission/runMode=local/1
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/1/cockroach-temp889184198
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/1/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/1
status:              initialized new cluster
clusterID:           70c0d4cc-68c8-4c8c-9ed1-e3e5502c72e7
nodeID:              1
CockroachDB node starting at 2018-01-29 20:11:49.362781374 +0000 UTC (took 0.2s)
build:               CCL v2.0-alpha.20180129-76-g7b3bf98 @ 2018/01/29 20:06:00 (go1.9.3)
admin:               http://127.0.0.1:37053
sql:                 postgresql://root@127.0.0.1:42135?application_name=cockroach&sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestDecommission/runMode=local/4
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/4/cockroach-temp296654668
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/4/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/4
status:              initialized new node, joined pre-existing cluster
clusterID:           70c0d4cc-68c8-4c8c-9ed1-e3e5502c72e7
nodeID:              2
CockroachDB node starting at 2018-01-29 20:11:49.367154574 +0000 UTC (took 0.2s)
build:               CCL v2.0-alpha.20180129-76-g7b3bf98 @ 2018/01/29 20:06:00 (go1.9.3)
admin:               http://127.0.0.1:44413
sql:                 postgresql://root@127.0.0.1:42231?application_name=cockroach&sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestDecommission/runMode=local/2
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/2/cockroach-temp049271720
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/2/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/2
status:              initialized new node, joined pre-existing cluster
clusterID:           70c0d4cc-68c8-4c8c-9ed1-e3e5502c72e7
nodeID:              3
CockroachDB node starting at 2018-01-29 20:11:49.397457527 +0000 UTC (took 0.2s)
build:               CCL v2.0-alpha.20180129-76-g7b3bf98 @ 2018/01/29 20:06:00 (go1.9.3)
admin:               http://127.0.0.1:39597
sql:                 postgresql://root@127.0.0.1:40429?application_name=cockroach&sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestDecommission/runMode=local/3
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/3/cockroach-temp290673985
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/3/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/3
status:              initialized new node, joined pre-existing cluster
clusterID:           70c0d4cc-68c8-4c8c-9ed1-e3e5502c72e7
nodeID:              4
CockroachDB node starting at 2018-01-29 20:11:59.893249759 +0000 UTC (took 0.7s)
build:               CCL v2.0-alpha.20180129-76-g7b3bf98 @ 2018/01/29 20:06:00 (go1.9.3)
admin:               http://127.0.0.1:33459
sql:                 postgresql://root@127.0.0.1:33505?application_name=cockroach&sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestDecommission/runMode=local/1
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/1/cockroach-temp739056031
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/1/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/1
status:              restarted pre-existing node
clusterID:           70c0d4cc-68c8-4c8c-9ed1-e3e5502c72e7
nodeID:              1
CockroachDB node starting at 2018-01-29 20:12:11.781718352 +0000 UTC (took 0.7s)
build:               CCL v2.0-alpha.20180129-76-g7b3bf98 @ 2018/01/29 20:06:00 (go1.9.3)
admin:               http://127.0.0.1:42893
sql:                 postgresql://root@127.0.0.1:34217?application_name=cockroach&sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestDecommission/runMode=local/2
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/2/cockroach-temp263435228
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/2/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/2
status:              restarted pre-existing node
clusterID:           70c0d4cc-68c8-4c8c-9ed1-e3e5502c72e7
nodeID:              3
CockroachDB node starting at 2018-01-29 20:12:23.55210533 +0000 UTC (took 0.6s)
build:               CCL v2.0-alpha.20180129-76-g7b3bf98 @ 2018/01/29 20:06:00 (go1.9.3)
admin:               http://127.0.0.1:34323
sql:                 postgresql://root@127.0.0.1:41087?application_name=cockroach&sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestDecommission/runMode=local/3
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/3/cockroach-temp889013800
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/3/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/3
status:              restarted pre-existing node
clusterID:           70c0d4cc-68c8-4c8c-9ed1-e3e5502c72e7
nodeID:              4
CockroachDB node starting at 2018-01-29 20:12:34.971507149 +0000 UTC (took 0.7s)
build:               CCL v2.0-alpha.20180129-76-g7b3bf98 @ 2018/01/29 20:06:00 (go1.9.3)
admin:               http://127.0.0.1:34273
sql:                 postgresql://root@127.0.0.1:33627?application_name=cockroach&sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestDecommission/runMode=local/1
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/1/cockroach-temp443003534
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/1/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/1
status:              restarted pre-existing node
clusterID:           70c0d4cc-68c8-4c8c-9ed1-e3e5502c72e7
nodeID:              1
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestDecommission_runMode=local558569193
--- FAIL: acceptance/TestDecommission (57.670s)
None
--- FAIL: acceptance/TestDecommission/runMode=local (57.670s)
test_log_scope.go:81: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestDecommission_runMode=local558569193
test_log_scope.go:62: use -show-logs to present logs inline
decommission_test.go:343: csv input:
	id,is_live,gossiped_replicas,is_decommissioning,is_draining
	1,false,20,true,false
	# 1 row
	id,is_live,gossiped_replicas,is_decommissioning,is_draining
	1,true,0,true,false
	# 1 row
	
	All target nodes report that they hold no more data. Please verify cluster health before removing the nodes.
	
	expected:
	[][]string{
	    {"id", "is_live", "gossiped_replicas", "is_decommissioning", "is_draining"},
	    {"1", "true", "0", "true", "true"},
	    {"# 1 row"},
	    {"All target nodes report that they hold no more data. Please verify cluster health before removing the nodes."},
	}
	errors:<nil>
	row #2, col #5: found "false" which does not match "true"
------- Stdout: -------
CockroachDB node starting at 2018-01-29 20:11:48.853669949 +0000 UTC (took 0.7s)
build:               CCL v2.0-alpha.20180129-76-g7b3bf98 @ 2018/01/29 20:06:00 (go1.9.3)
admin:               http://127.0.0.1:42815
sql:                 postgresql://root@127.0.0.1:41853?application_name=cockroach&sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestDecommission/runMode=local/1
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/1/cockroach-temp889184198
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/1/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/1
status:              initialized new cluster
clusterID:           70c0d4cc-68c8-4c8c-9ed1-e3e5502c72e7
nodeID:              1
CockroachDB node starting at 2018-01-29 20:11:49.362781374 +0000 UTC (took 0.2s)
build:               CCL v2.0-alpha.20180129-76-g7b3bf98 @ 2018/01/29 20:06:00 (go1.9.3)
admin:               http://127.0.0.1:37053
sql:                 postgresql://root@127.0.0.1:42135?application_name=cockroach&sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestDecommission/runMode=local/4
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/4/cockroach-temp296654668
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/4/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/4
status:              initialized new node, joined pre-existing cluster
clusterID:           70c0d4cc-68c8-4c8c-9ed1-e3e5502c72e7
nodeID:              2
CockroachDB node starting at 2018-01-29 20:11:49.367154574 +0000 UTC (took 0.2s)
build:               CCL v2.0-alpha.20180129-76-g7b3bf98 @ 2018/01/29 20:06:00 (go1.9.3)
admin:               http://127.0.0.1:44413
sql:                 postgresql://root@127.0.0.1:42231?application_name=cockroach&sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestDecommission/runMode=local/2
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/2/cockroach-temp049271720
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/2/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/2
status:              initialized new node, joined pre-existing cluster
clusterID:           70c0d4cc-68c8-4c8c-9ed1-e3e5502c72e7
nodeID:              3
CockroachDB node starting at 2018-01-29 20:11:49.397457527 +0000 UTC (took 0.2s)
build:               CCL v2.0-alpha.20180129-76-g7b3bf98 @ 2018/01/29 20:06:00 (go1.9.3)
admin:               http://127.0.0.1:39597
sql:                 postgresql://root@127.0.0.1:40429?application_name=cockroach&sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestDecommission/runMode=local/3
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/3/cockroach-temp290673985
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/3/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/3
status:              initialized new node, joined pre-existing cluster
clusterID:           70c0d4cc-68c8-4c8c-9ed1-e3e5502c72e7
nodeID:              4
CockroachDB node starting at 2018-01-29 20:11:59.893249759 +0000 UTC (took 0.7s)
build:               CCL v2.0-alpha.20180129-76-g7b3bf98 @ 2018/01/29 20:06:00 (go1.9.3)
admin:               http://127.0.0.1:33459
sql:                 postgresql://root@127.0.0.1:33505?application_name=cockroach&sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestDecommission/runMode=local/1
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/1/cockroach-temp739056031
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/1/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/1
status:              restarted pre-existing node
clusterID:           70c0d4cc-68c8-4c8c-9ed1-e3e5502c72e7
nodeID:              1
CockroachDB node starting at 2018-01-29 20:12:11.781718352 +0000 UTC (took 0.7s)
build:               CCL v2.0-alpha.20180129-76-g7b3bf98 @ 2018/01/29 20:06:00 (go1.9.3)
admin:               http://127.0.0.1:42893
sql:                 postgresql://root@127.0.0.1:34217?application_name=cockroach&sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestDecommission/runMode=local/2
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/2/cockroach-temp263435228
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/2/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/2
status:              restarted pre-existing node
clusterID:           70c0d4cc-68c8-4c8c-9ed1-e3e5502c72e7
nodeID:              3
CockroachDB node starting at 2018-01-29 20:12:23.55210533 +0000 UTC (took 0.6s)
build:               CCL v2.0-alpha.20180129-76-g7b3bf98 @ 2018/01/29 20:06:00 (go1.9.3)
admin:               http://127.0.0.1:34323
sql:                 postgresql://root@127.0.0.1:41087?application_name=cockroach&sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestDecommission/runMode=local/3
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/3/cockroach-temp889013800
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/3/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/3
status:              restarted pre-existing node
clusterID:           70c0d4cc-68c8-4c8c-9ed1-e3e5502c72e7
nodeID:              4
CockroachDB node starting at 2018-01-29 20:12:34.971507149 +0000 UTC (took 0.7s)
build:               CCL v2.0-alpha.20180129-76-g7b3bf98 @ 2018/01/29 20:06:00 (go1.9.3)
admin:               http://127.0.0.1:34273
sql:                 postgresql://root@127.0.0.1:33627?application_name=cockroach&sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestDecommission/runMode=local/1
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/1/cockroach-temp443003534
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/1/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster563015476/1
status:              restarted pre-existing node
clusterID:           70c0d4cc-68c8-4c8c-9ed1-e3e5502c72e7
nodeID:              1
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestDecommission_runMode=local558569193

```

Please assign, take a look and update the issue accordingly.

--- a/pkg/acceptance/decommission_test.go
+++ b/pkg/acceptance/decommission_test.go
@@ -227,7 +227,8 @@ func testDecommissionInner(
 
 		exp := [][]string{
 			decommissionHeader,
-			{strconv.Itoa(int(target)), "true", "0", "true", "true"},
+			// Note that the process may return before the node is known as draining.
+			{strconv.Itoa(int(target)), "true", "0", "true", "false|true"},
 			decommissionFooter,
 		}
 		if err := matchCSV(o, exp); err != nil {
@@ -284,7 +285,8 @@ func testDecommissionInner(
 
 		exp := [][]string{
 			decommissionHeader,
-			{strconv.Itoa(int(target)), "true", "0", "true", "true"},
+			// Note that the process may return before the node is known as draining.
+			{strconv.Itoa(int(target)), "true", "0", "true", "false|true"},
 			decommissionFooter,
 		}
 		if err := matchCSV(o, exp); err != nil {
@@ -331,7 +333,8 @@ func testDecommissionInner(
 
 		exp := [][]string{
 			decommissionHeader,
-			{strconv.Itoa(int(target)), "true", "0", "true", "true"},
+			// Note that the process may return before the node is known as draining.
+			{strconv.Itoa(int(target)), "true", "0", "true", "false|true"},
 			decommissionFooter,
 		}
 		if err := matchCSV(o, exp); err != nil {

